db/config.cc

```diff
@@ -1361,6 +1361,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     // alternator_max_items_in_batch_write matches DynamoDB behaviour of size limit, but with different value - for DynamoDB it's 25
     // (see DynamoDB's documentation for BatchWriteItem command)
     , alternator_max_items_in_batch_write(this, "alternator_max_items_in_batch_write", value_status::Used, 100, "Maximum amount of items in single BatchItemWrite call.")
+    , alternator_describe_table_info_timeout_in_seconds(this, "alternator_describe_table_info_timeout_in_seconds", liveness::LiveUpdate, value_status::Used, 60 * 60 * 6,
+        "The timeout for DescribeTable information - table size in bytes. This is how long calculated value will be reused before recalculation.")
```

Comment:
I think the name "timeout" is confusing. It's not a "timeout", rather it's an amount of time where the value is cached, or equivalently - the amount of time between updates.
We have other configuration names that describe cache times called permissions_validity_in_ms, permissions_update_interval_in_ms, repair_hints_batchlog_flush_cache_time_in_ms, wasm_cache_timeout_in_ms, sstable_compression_dictionaries_retrain_period_in_seconds. Sadly each one invented a different naming scheme :-(

By the way, most of them used milliseconds, not seconds. The nice thing about milliseconds is that a test can make it much lower than one whole second.